### PR TITLE
Vacms 16382 metatag updates

### PR DESCRIPTION
**Pull request by alexfinnarn**
## Description

closes #16382 

## Testing done

1. Manual testing as described in the QA steps.
2. Updated snapshot tests to incorporate "lastUpdated" field.

## Screenshots

Custom tags for "/central-iowa-health-care/events/52265"
Screenshot 2023-12-29 at 2 01 41 PM

Default tags:
Screenshot 2023-12-29 at 2 02 14 PM

## QA steps

1. Go to `/central-iowa-health-care/events/52265` to load the page.
2. Look at the metatag output and see that it matches the screenshot for custom tags.
3. Uncomment the `/central-iowa-health-care/events/52265` URL in the `urlsForBanner` variable in the `src/templates/globals/meta/index.tsx` file to show the iOS banner.
4. See that the iOS metatags now show up. 
5. Change `resource.metatags?.length > 0` to `resource.metatags?.length > 1000` in `src/templates/globals/meta/index.tsx` to get the default metatags to show up. 
6. Compare the metatags you see to the default metatag output screenshot.
7. For the two images added, make sure they resolve at the paths: "/img/design/logo/va-og-twitter-image.png" and "/img/design/logo/va-og-image.png".

```[tasklist]
- [ ] Automated tests have passed
- [ ] Tugboat environment was generated without errors
```

### CODE REVIEW

File: src/data/queries/event.ts [hunk 1]
```diff
@@ -71,5 +71,6 @@ export const formatter: QueryFormatter<NodeEvent, Event> = (
     description: entity.field_description,
     link: entity.field_link,
     urlOfOnlineEvent: entity.field_url_of_an_online_event,
+    lastUpdated: entity.field_last_saved_by_an_editor || entity.created,
```
Comment (tjheffner) [inline src/data/queries/event.ts:74]: move this inside of the object from `...entityBaseFields`,  will save repeating

File: src/types/formatted/event.ts [hunk 2]
```diff
@@ -52,4 +52,5 @@ export type Event = PublishedEntity & {
   description: string
   link: Link | null
   urlOfOnlineEvent: urlOfOnlineEvent
+  lastUpdated: string
```
Comment (tjheffner) [inline src/types/formatted/event.ts:55]: add `lastUpdated` to `PublishedEntity`, will save repeating

**Review by tjheffner — COMMENTED** [review]
Overall looks great, works great. Just two minor nits on where the data/types get added to save us repeating in the long run

**Comment by alexfinnarn** [comment]
@tjheffner I had to add the `lastUpdated: entity.field_last_saved_by_an_editor || entity.created` line a couple of other places where `...entityBaseFields` could be used, if appropriate, but I didn't want to make that assumption. I think I've updated the code and tests to reflect the type changes.

**Review by tjheffner — APPROVED** [review]

**Pull request merged**